hurl/admin.hurl

```diff
@@ -0,0 +1,12 @@
+# Get the admin homepage
+GET {{host}}/admin
```

Comment:
This is fine for now, but at some point these tests will break because _origin_ breaks. The solution would be to inject a mock Changelog origin as the server upstream of the cache.

Reply:
This is an excellent point @jamesarosen. Ie touches on this commit comment:

> There is one big issue with this. Can you spot it?

You are absolutely right: the big variable here is the origin, which we should control directly. Right now, we have this huge external dependency - the changelog.com origin - and also we are testing the **deployed** version, not the version in the PR.

The biggest issue is that we are currently **not** testing the version in the PR. The `{{host}}` variable was the hint. The second related issue is that we should be in control of the origin part of the test suite, rather than depending on an external service. We both know that this will result in side effects, and result in a brittle test suite.

In the context of this PR, this is fine, but we should definitely follow-up soon with the improvements mentioned above.

Great catch! 🧤